src/transmission_manager.rs

```diff
@@ -26,14 +31,15 @@ define_person_property_with_default!(
     InfectiousStatus::Susceptible
 );
 
-fn evaluate_transmission(context: &mut Context, contact_id: PersonId, _transmittee_id: PersonId) {
-    // for now, we assume all transmission events are sucessful
-    // we have the contact_id as an argument to this fcn so that person-person pair transmission potential
-    // based on interventions can be evaluated to determine whether this transmission event is actually successful
-    if matches!(
-        context.get_person_property(contact_id, InfectiousStatusType),
-        InfectiousStatus::Susceptible
-    ) {
+fn evaluate_transmission(context: &mut Context, contact_id: PersonId, _transmitter_id: PersonId) {
+    // For now, we assume all transmission events are sucessful.
+    // However, in the future, the success of a transmission event may depend on person-level interventions,
+    // such as whether an agent is wearing a mask. For this reason, we pass the transmitter as well to this
+    // function. This will allow us to evaluate the success of a transmission event based on the properties of
+    // both the transmitter and the contact in the future.
```

Comment:
Shouldn't this comment be above the function? And explain what the function does first.